src/pages/EditMonitor.vue

```diff
@@ -246,19 +234,79 @@
                             </template>
 
                             <!-- Hostname -->
-                            <!-- TCP Port / Ping / DNS / Steam / MQTT / Radius / Tailscale Ping only -->
-                            <div v-if="monitor.type === 'port' || monitor.type === 'ping' || monitor.type === 'dns' || monitor.type === 'steam' || monitor.type === 'gamedig' ||monitor.type === 'mqtt' || monitor.type === 'radius' || monitor.type === 'tailscale-ping'" class="my-3">
-                                <label for="hostname" class="form-label">{{ $t("Hostname") }}</label>
+                            <!-- TCP Port / Ping / DNS / Steam / MQTT / Radius / Tailscale Ping / SNMP only -->
+                            <div v-if="monitor.type === 'port' || monitor.type === 'ping' || monitor.type === 'dns' || monitor.type === 'steam' || monitor.type === 'gamedig' || monitor.type === 'mqtt' || monitor.type === 'radius' || monitor.type === 'tailscale-ping' || monitor.type === 'snmp'" class="my-3">
+                                <label for="hostname" class="form-label">{{ $t("Hostname or IP Address") }}</label>
```

Comment:
nit: I don't think this translation does make sense for every monitor here.
F.ex. we currently don't support IP-adresses for the dns monitor.

Likely does not matter in the real world.

Reply:
Probably best for me to put it back to simply "Hostname" to limit number of potentially disruptive frontend changes from this PR. My intention was to avoid potential confusion from people not realizing it is also an "IP address" field.

Reply:
Resolved with e237d66bfcea117cb405395f6ff9a39c9f8b2cce